src/spreddit.py

```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8 -*-
 
-from instagram import Instagram, ProcessingError, InstagramLoginError
-from reddit import Reddit            
+from .instagram import Instagram, ProcessingError, InstagramLoginError
```

Comment:
from .instagram import Instagram
from .errors import ProcessingError, InstagramLoginError

Reply:
I'd update that